Source/Notification/CallNotificationCenter.swift

```diff
@@ -21,86 +21,72 @@
 import Foundation
 
 /// The CallNotificationCenter class is used to add & remove call observer.
-public class CallNotificationCenter {
+open class CallNotificationCenter {
```

Comment:
If it's not for swift3.0 migration, I suggest that we don't change these access type.